package.json

```diff
@@ -34,8 +34,8 @@
     "lint-staged": "^6.0.0",
     "mocha": "^4.0.1",
     "prettier": "^1.11.1",
-    "typescript": "~2.8.1",
-    "typescript-eslint-parser": "^15.0.0"
+    "typescript": "~2.9",
+    "typescript-eslint-parser": "^16.0.1"
```

Comment:
I checked this out locally to see how far we could increase the version of typescript + the parser before more tests started breaking, and it looks like we can upgrade to the following safely from this commit.

```
"typescript": "~3.0.3",
"typescript-eslint-parser": "^19.0.2"
```

Should we do that in this PR or a follow up? /cc @bradzacher